pkg/logger/types.go

```diff
@@ -32,4 +32,5 @@ type LogRequest struct {
 	Component        string
 	Endpoint         string
 	Metadata         map[string][]string
+	TlsCertName      string
```

Comment:
Is it a server certificate? so I think certName is ok.

```suggestion
	certName      string
```